admin/class-metabox.php

```diff
@@ -799,4 +839,17 @@ public function scoring_svg() {
 				</svg>
 			</script>';
 	}
+
+	/**
+	 * Returns a json object with scraper data.
+	 *
+	 * @return array
+	 */
+	private function get_scraper_json() {
+		$file = plugin_dir_path( WPSEO_FILE ) . 'languages/wordpress-seo-' . get_locale() . '.json';
```

Comment:
@omarreiss I don't know if this works on all server configurations, this was just something I quickly drafted to check if the translations were working. I think we cannot read this json on all possible servers.